### PR TITLE
Normalized Pixel Variation AOV wrt the user noise threshold

### LINKED ISSUE #2404: Normalize Pixel Variation AOV wrt. the user's noise threshold
Divide computed variation by the noise threshold set by the user such that AOV = 1 means variation = noise threshold.

### PR DESCRIPTION
**Pull request by kushWithoutWax**
Fixes #2404 

Issue: After applying the post-processing stage, the image before and after this change look the same.

### CODE REVIEW

File: src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp [hunk 1]
```diff
@@ -552,7 +552,7 @@ namespace
                         {
                             Color3f variation;
                             m_variation_aov_tile->get_pixel(pt.x, pt.y, variation);
-                            variation[0] += pb.m_block_error;
+                            variation[0] += pb.m_block_error / m_params.m_noise_threshold;
```
Comment (oktomus) [inline src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: What if the noise threshold is zero ?

Comment (kushWithoutWax) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: The noise threshold has a minimum value of 0.0001 . It cannot be set to 0 either in the .appleseed file or through appleseed studio.
https://github.com/appleseedhq/appleseed/blob/127bbc39b2bd9a4cbcaff2b94552757aec6f8058/src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp#L1014 ensures it I think.

Comment (oktomus) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: Ok nice

Comment (oktomus) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: Ok nice

Comment (oktomus) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: Could you share some images of what this change does ?

Comment (kushWithoutWax) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: Screenshot from 2019-04-05 07-16-09
The scene 
NormBuild
Using thresholds of 0.004, 0.3, 0.6
NormalWithoutPost
Without the post processing stage

The problem with this is that after the post processing stage, the aov image with my changes on and those without are exactly the same. I am trying to figure out what is causing the problem

Comment (oktomus) [reply src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp]: ok. I think i already tried this when I first implented it and it wasnt that great. Though I was normalizing while rendering whereas here you normalize it only for the final aov.

**Pull request closed without merging**